rook/scripts/zap-disk-auto.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+# THIS WILL ZAP DISKS WITHOUT REQUIRING CONFIRMATION - USE `zap-disk.sh` INSTEAD
+# DON'T RUN THIS
+
+set -e -o pipefail
+
+here="$(dirname "$(readlink -f "$0")")"
+
+if [[ "$#" -lt 1 || "$#" -gt 2 ]]; then
+    echo "${0} <host ip> [sdb]"
```

Comment:
Just to make sure it get's printed and not lost in some buffer somewhere

```suggestion
    echo "${0} <host ip> [sdb]" >&2
```

Same on other echo before a `exit 1`